pkg/kube/client.go

```diff
@@ -325,10 +325,8 @@ func (c *Client) Delete(resources ResourceList) (*Result, []error) {
 		return nil
 	})
 	if err != nil {
-		// Rewrite the message from "no objects visited" if that is what we got
-		// back
 		if err == ErrNoObjectsVisited {
-			err = errors.New("object not found, skipping delete")
+			err = fmt.Errorf("object not found, skipping delete: %w", ErrNoObjectsVisited)
```

Comment:
Shouldn't this be:

```suggestion
			err = fmt.Errorf("object not found, skipping delete: %w", err)
```
in case err has any additional context?

Reply:
Hi @joejulian ,
Thanks for the comment! This ErrNoObjectsVisited is customized one, and at the moment the usage for this error has no additional context appended, but I agreed with you, by directly return this err itself make it more generic. Please check my update.